.drone.yml

```diff
@@ -0,0 +1,20 @@
+kind: pipeline
+name: default
+
+platform:
+  os: linux
+  arch: arm
+
+steps:
+- name: build
+  image: golang
```

Comment:
Any reason the image is for Go?

Reply:
Yeah, when I tried switching away from Go, to, say Java images, I would get

```
standard_init_linux.go:190: exec user process caused "exec format error"
```

I then presumed that the `.go` in the above might be referring to Go, and that some Drone script needed Go to be available. So I just gave up and wend with the `golang` image.